src/equations/2d/ideal_glm_mhd.jl

```diff
@@ -415,14 +415,21 @@ function flux_hll(u_ll, u_rr, orientation, equation::IdealGlmMhdEquations2D)
   return SVector(f1, f2, f3, f4, f5, f6, f7, f8, f9)
 end
 
-
-# strong form of nonconservative flux on a side, e.g., the Powell term
-#     phi^L 1/2 (B^L+B^R) normal - phi^L B^L normal = phi^L 1/2 (B^R-B^L) normal
-# OBS! 1) "weak" formulation of split DG already includes the contribution -1/2(phi^L B^L normal)
-#         so this routine only adds 1/2(phi^L B^R normal)
-#         analogously for the Galilean nonconservative term
-#      2) this is non-unique along an interface! normal direction is super important
-function noncons_interface_flux(u_left, u_right, orientation, equation::IdealGlmMhdEquations2D)
+"""
+    noncons_interface_flux(u_left, u_right, orientation, mode, equation::IdealGlmMhdEquations2D)
+    
+Strong form of nonconservative flux on a side (Powell and GLM terms)
+     phi^L 1/2 (B^L+B^R) normal - phi^L B^L normal = phi^L 1/2 (B^R-B^L) normal
+OBS! 1) The non-conservative interface flux depends on the discretization. Following "modes" are available:
+        * :weak: 'weak' formulation of split DG already includes the contribution -1/2(phi^L B^L normal)
+                so this mode only adds 1/2(phi^L B^R normal)
+                ... analogously for the Galilean nonconservative term
+        * :whole: This mode adds the whole non-conservative term: phi^L 1/2 (B^R-B^L)
+        * :inner: This mode adds the split-form DG volume integral contribution: This is equivalent to (2)-(1)
+                - 1/2(phi^L B^L)
+     2) this is non-unique along an interface! normal direction is super important
+"""
```

Comment:
````suggestion
"""
    noncons_interface_flux(u_left, u_right, orientation, mode, equation::IdealGlmMhdEquations2D)

Strong form of nonconservative flux on a side (Powell and GLM terms)
```math
phi^L 1/2 (B^L + B^R)_{normal} - phi^L B^L+{normal} = phi^L 1/2 (B^R - B^L)_{normal}
```
!!! note
    The non-conservative interface flux depends on the discretization. Following "modes" are available:
    * `:weak`: 'weak' formulation of split DG already includes the contribution
      ``-1/2 (phi^L B^L_{normal})`` so this mode only adds ``1/2 (phi^L B^R_{normal})``,
      analogously for the Galilean nonconservative term
    * `:whole`: This mode adds the whole non-conservative term: phi^L 1/2 (B^R-B^L)
    * `:inner`: This mode adds the split-form DG volume integral contribution: This is equivalent to
      ``(2)-(1) - 1/2 (phi^L B^L)``
!!! warning
    This is non-unique along an interface! The normal direction is super important.
"""
````
This makes the rendering of the docstring useful in the REPL. Check
```julia
help?> Trixi.noncons_interface_flux
```
Otherwise, this PR looks good to me :+1: